tests/langchain/test_langchain_autolog.py

```diff
@@ -785,6 +789,37 @@ def test_langchain_autolog_callback_injection_in_invoke(invoke_arg, generate_cal
         assert callbacks[0].logs == ["chain_start", "chain_end"]
 
 
+@pytest.mark.parametrize("invoke_arg", ["args", "kwargs"])
+@pytest.mark.parametrize(
+    "generate_callbacks",
+    [lambda: [CustomCallbackHandler()], lambda: BaseCallbackManager([CustomCallbackHandler()])],
```

Comment:
can we also test `AsyncCallbackHandler`?

Reply:
added test!